cdl.bib

```diff
@@ -198,27 +158,22 @@ @article{LowEtal20
 
 @article{StewVelu21,
 	author = {R Stewart and S Velupillai},
-	date-added = {2021-10-26 21:34:07 -0400},
-	date-modified = {2021-10-26 21:38:24 -0400},
 	journal = {Neuropsychopharmacology},
 	pages = {252--253},
 	title = {Applied natural language processing in mental health big data},
 	volume = {46},
 	year = {2021}}
 
-@article{CoppEtal17,
+@inproceedings{CoppEtal17,
 	author = {G Coppersmith and C Hilland and O Frieder and R Leary},
-	date-added = {2021-10-26 21:32:51 -0400},
-	date-modified = {2021-10-26 21:34:02 -0400},
-	journal = {{IEEE} {Xplore}},
-	pages = {doi.org/10.1109/BHI.2017.7897288},
+	booktitle = {2017 {IEEE} {EMBS} International Conference on Biomedical \& Health Informatics {(BHI})},
```

Comment:
is the {(BHI}) part correct?

Reply:
it's "correct" in that it passses the checker  -- I opened #53 about this. 

I agree that I'd expect it to be `({BHI})` instead, and the checker should probably be fixed to accept this, but since the difference shouldn't actually affect the output, I went ahead and formatted as the checker wanted for now.

Reply:
:+1: